packages/defender-sdk/src/index.ts

```diff
@@ -48,46 +51,56 @@ export class Defender {
     // support for using relaySigner from Defender Actions
     this.actionCredentials = options.credentials;
     this.actionRelayerArn = options.relayerARN;
+    this.httpsAgent = options.httpsAgent;
   }
 
   public networks(opts?: ListNetworkRequestOptions): Promise<Network[]> {
-    return getClient(NetworkClient, { apiKey: this.apiKey, apiSecret: this.apiSecret }).listSupportedNetworks(opts);
+    return getClient(NetworkClient, {
+      apiKey: this.apiKey,
+      apiSecret: this.apiSecret,
+      httpsAgent: this.httpsAgent,
```

Comment:
We probably should allow clients to set timeout option too if needed. Wdyt?

Reply:
Yep, by exposing the httpsAgent we allow them to set any policy they need, same example including timeout:

```js
const client = new Defender({ 
    apiKey: process.env.API_KEY,
    apiSecret: process.env.API_SECRET, 
    httpsAgent: https.Agent({ 
       keepAlive: true,
       timeout: 200000,
   }) 
});
```

Reply:
Hmm interesting, thought timeout config is outside of httpsAgent initiation